docs/user/corrections/transmission.rst

```diff
@@ -0,0 +1,109 @@
+.. _user.corrections.transmission:
+
+Transmission
+============
+
+The measured intensity must be corrected to account for sample transmission:
+
+.. math::
+
+    I'_{sample}(x,y,\lambda) = \frac{I_{sample}(x,y,\lambda)}{T(\lambda,x,y)}
+
+The transmission correction is calculated using a transmission run and an empty beam transmission
+run (reference).
+It is possible to specify the transmission value directly in the transmission parameter ``"value"``,
+however, this is mostly used for diagnostic purposes.
+
+.. code-block:: json
+
+    {
+      "sample": {
+        "runNumber": None,
+        "loadOptions": {},
+        "thickness": 1.0,
+        "transmission": {
+          "runNumber": None,
+          "value": None,
+          "errorTolerance": None
+        }
+      },
+      "background": {
+        "runNumber": None,
+        "transmission": {
+          "runNumber": None,
+          "value": None
+        }
+      },
+      "emptyTransmission": {
+        "runNumber": None,
+        "value": None
+      },
+      "mmRadiusForTransmission": None,
+      "useTimeSliceTransmission": False,
+      "useThetaDepTransCorrection": True,
+    }
+
+Time Slice Transmission (Bio-SANS only)
+---------------------------------------
+
+When using time slicing (``"useTimeSlice": True``), users can optionally calculate the transmission
+correction using the time sliced sample run by setting the parameter
+``"useTimeSliceTransmission": True``. The sample transmission run number is ignored when
+``"useTimeSliceTransmission": True``. The time slice transmission option can be used when the sample
+transmission is expected to change over time.
+
+Time slices for which the transmission calculation fails will be skipped. The transmission
+calculation can fail due to all transmission values being NaN or if the transmission error is
+higher than the allowed relative transmission error, which is configurable in the sample
+transmission parameter ``"errorTolerance"`` (default: 0.01).
```

Comment:
Insert a use-case when error in the transmission is bigger than the tolerance. The one we're dealing with is when the span of the time slice are so small that few counts can be collected when calculating the transmission, leading to large statistical errros.